tests/test_documents.py

```diff
@@ -10,10 +10,15 @@
 
 
 def test_upload_document(requests_mock):
-    mock_upload = requests_mock.post("https://api.eu.onfido.com/v3.3/documents/", json=[])
+    mock_upload = requests_mock.post("https://api.eu.onfido.com/v3.4/documents/", json=[])
 
     request_body = {"applicant_id": fake_uuid,
-                    "document_type": "driving_licence"}
+                    "document_type": "driving_licence",
+                    "location": {
+                        "ip_address": "127.0.0.1",
+                        "country_of_residence": "GBR"
+                        }
+                    }
```

Comment:
This seems to have bad indentation.
Should it be like this?
```
{
  "applicant_id": fake_uuid,
  "document_type": "driving_licence",
  "location": {
    "ip_address": "127.0.0.1",
    "country_of_residence": "GBR"
  }
)
```

Reply:
Yeah, you're right. Four spaces instead of two as habit. I'm going to fix that :-)